tests/resources/synthetic_001/synthetic_001.json

```diff
@@ -7,11 +7,28 @@
     "camera_model": "colour-science",
     "iso": 100,
     "temperature": 6500,
-    "additional_camera_settings": null,
+    "additional_camera_settings": "",
```

Comment:
**Major**

I haven't checked but we will want to update the JSON schema accordingly.

Reply:
happy either way to be honest as None is also a valid json serialisable, whilst this was proably a copy and paste / auto complete while doing the ProjectSettings i was thinking

string value when empty is "" vs an instance of an object which would be None

Happy to change whichever